internal/controller/standalone_pgadmin/pod.go

```diff
@@ -303,17 +311,17 @@ loadServerCommand
 	var reloadScript = `
 exec {fd}<> <(:)
 while read -r -t 5 -u "${fd}" || true; do
-	if [ "${cluster_file}" -nt "/proc/self/fd/${fd}" ] && loadServerCommand
-	then
-		exec {fd}>&- && exec {fd}<> <(:)
-		stat --format='Loaded shared servers dated %y' "${cluster_file}"
-	fi
-	if [ ! -d /proc/$(cat $PGADMIN4_PIDFILE) ]
-	then
-		pgadmin4 &
-		echo $! > $PGADMIN4_PIDFILE
-		echo "Restarting pgAdmin4"
-	fi
+    if [ "${cluster_file}" -nt "/proc/self/fd/${fd}" ] && loadServerCommand
+    then
+        exec {fd}>&- && exec {fd}<> <(:)
+        stat --format='Loaded shared servers dated %y' "${cluster_file}"
+    fi
+    if [ ! -d /proc/$(cat $PGADMIN4_PIDFILE) ]
+    then
+    gunicorn -c /etc/pgadmin/gunicorn_config.py pgAdmin4:app &
```

Comment:
Why do we not need the `--chdir` flag here? :thinking:

Reply:
We do. I missed that change during the refactor. Thanks for the catch; I'll update this.

Reply:
Could also consider pulling that out into a function like we did with `loadServerCommand`, but not a blocker